# Goal
---
# Result
aist_aruco_msgs/PointCorrespondenceArray[] correspondences_set
string                                     message
---
# Feedback
